libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -15,17 +15,35 @@ import {
 })
 export class FieldsService {
   private fields = {
-    publisher: new SimpleSearchField('OrgForResource', 'asc', this.injector),
+    publisher: new SimpleSearchField(
+      'OrgForResourceObject.default',
+      'asc',
+      this.injector
+    ),
     format: new SimpleSearchField('format', 'asc', this.injector),
     publicationYear: new SimpleSearchField(
       'publicationYearForResource',
       'desc',
       this.injector
     ),
-    topic: new TopicSearchField(this.injector),
+    topic: new SimpleSearchField('cl_topic.default', 'asc', this.injector),
```

Comment:
Why this change ? What was the issue ?

Reply:
I used this aggregation to stay consistent with the INSPIRE keywords aggregation 
```
    inspireKeyword: new SimpleSearchField(
      'th_httpinspireeceuropaeutheme-theme_tree.default',
      'asc',
      this.injector
    ),
```

Reply:
Same as my comment above, I think we should exclude this change from the current PR as it will definitely break things elsewhere.

Reply:
as with the previous comment, I can revert this to what it was and amend the commit :smile: